tests/test_structs.py

```diff
@@ -136,3 +136,61 @@ def test_frozen_trial_repr():
                         intermediate_values={})
 
     assert trial == eval(repr(trial))
+
+
+def test_study_summary_study_id():
+    # type: () -> None
+
+    study = optuna.create_study()
+    summaries = study._storage.get_all_study_summaries()
+    assert len(summaries) == 1
+
+    summary = summaries[0]
+
+    assert summary.study_id == summary._study_id
```

Comment:
Ditto regarding explicitly ignoring the warning. Could you take a look at my previous comment?

Reply:
Fixed: https://github.com/optuna/optuna/pull/718/commits/94e802a35dbf3e99fdb60706787bfd42a1604968